### ISSUE TITLE
kubectl get clusters does not work consistently

### ISSUE THREAD

**Issue by johnbelamaric** [nephio-project/nephio#327]
The problem is this:

```
ubuntu@nephio-r1-e2e:~$ kubectl api-resources | grep clusters
clusters                          cl           cluster.x-k8s.io/v1beta1                  true         Cluster
clusters                                       clusterregistry.k8s.io/v1alpha1           true         Cluster
clusterselectors                               configmanagement.gke.io/v1                false        ClusterSelector
workloadclusters                               infra.nephio.org/v1alpha1                 true         WorkloadCluster
dockerclusters                                 infrastructure.cluster.x-k8s.io/v1beta1   true         DockerCluster
ubuntu@nephio-r1-e2e:~$ 
```

At some point, the `clusterregistry.k8s.io/v1alpha1` started interfering with the `cluster.x-k8s.io/v1beta1`. Where did the cluster registry one come from? I never saw this until we update the mgmt cluster to a newer K8s version. I don't know if that's because the algorithm in the older k8s version always picked the right "clusters", or because something else changed and we now *have* the clusterregistry one and we *did not* before. It needs to be investigated - do we even *need* the clusterregistry one? How is it getting added to our system?

Most of the e2e scripts have been fixed to account for this, but it will still cause confusion for end users if we do not fix it.

/kind bug

**Comment by electrocucaracha** [comment]
I could be wrong, but it doesn't come from [ConfigSync](https://github.com/GoogleContainerTools/kpt-config-sync/blob/v1.13/pkg/kinds/kinds.go#L144-L147)?

**Comment by johnbelamaric** [comment]
I don't think so. That's more an informational thing in CS for when it is running on clusters that have that installed. Let me grep around.

**Comment by johnbelamaric** [comment]
Oof, I think you're right: 

https://github.com/GoogleContainerTools/kpt-config-sync/blob/v1.13/manifests/cluster-registry-crd.yaml

**Comment by johnbelamaric** [comment]
Let me look into why that's there. It's probably for ClusterSelectors which we do not use.

**Comment by johnbelamaric** [comment]
Ok, I filed an issue with CS and also reached out to some folks internally.
